Apps/MAPL_GridCompSpecs_ACG.py

```diff
@@ -79,10 +79,7 @@ def internal_name(name):
             return name
     @staticmethod
     def mangled_name(name):
-        if name[-1] == '*':
-            return "'" + name[:-1] + "'//trim(comp_name)"
-        else:
-            return "'" + name + "'"
+        return "'" + name.replace("*","'//trim(comp_name)//'") + "'"
```

Comment:
Ah - it was already relying on the dynamic name.     Can you post the resulting declare pointers include file to this ticket?